client/controllers/editElectionCtrl.js

```diff
@@ -139,5 +139,21 @@ module.exports = [
         .path('/admin/create_election')
         .search({ election: JSON.stringify(election) });
     };
+
+    $scope.copyToClipboard = function (text) {
```

Comment:
> gimmick to copy the access code with an icon/button

😎 Cool. Kinda `lol` code, but I'll allow it 👍🏻

Reply:
Yeee, but it is the way to do it :laughing:.